client/my-sites/plugins/plugin-icon/style.scss

```diff
@@ -4,20 +4,12 @@
 
 .plugin-icon {
 	float: left;
-	margin-right: 16px;
-	width: 56px;
-	height: 56px;
+	margin-right: 12px;
+	width: 48px;
+	height: 48px;
```

Comment:
@iamtakashi The `PluginIcon` component is used at many places and at some of them, the style changes are not an improvement and most likely unintentional. The `/plugins/akismet` page looks like this in `master`:

<img width="501" alt="plugin-meta-production" src="https://user-images.githubusercontent.com/664258/29862167-12cd0e66-8d6c-11e7-8168-d8dd9215e71f.png">

The icon is large (56x56px) and has the same spacing (24px) on all 4 sides.

This is the new look in our branch:

<img width="502" alt="plugin-meta-branch" src="https://user-images.githubusercontent.com/664258/29862203-31fc47fc-8d6c-11e7-9ca2-bc2e26b5843a.png">

Small icon, uneven spacing. If your intent was to improve the styling only for `.plugin-item` and `.plugins-browser-item`, let's scope the CSS rules accordingly.

Reply:
My bad. I'll fix this also.